utils/lzbench/Makefile

```
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2025 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lzbench
+PKG_VERSION:=2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/inikep/lzbench.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=8147278a3322ddfa193de589707a2627c99b3183267e7a1cda7b8a2628edc54a
```

Is there any specific reason, why to you Git clone instead of downloading tagged Release?


```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
PKG_SOURCE_URL:=https://codeload.github.com/inikep/lzbench/tar.gz/v$(PKG_VERSION)?
PKG_HASH:=f86186864d4de6fdb187e34ddb5426f9c4910861726413fcba55eae65ef5a25b
```